cgo/cgo.go

```diff
@@ -115,6 +116,8 @@ var cgoAliases = map[string]string{
 	"C.uint32_t":  "uint32",
 	"C.uint64_t":  "uint64",
 	"C.uintptr_t": "uintptr",
+	"C.float":     "float32",
+	"C.double":    "float64",
```

Comment:
I wasn't aware that these were necessary. Is there code that uses `C.float` and `C.double` instead of simply `float32` and `float64`? So far, using floats in testdata/cgo seems to have worked well so far, so I'm not sure what this fixes exactly. Note that a C `float` is already an alias for `float32` and C `double` is already an alias for `float64`.

In any case, these should really be done in a separate PR or a cleanly separated commit, as this is unrelated to `LDFLAGS` support.

Reply:
Well it does work fine for tinygo because the types can be mapped directly. On go you need to convert the types so usually the cgo codes use that C.float or C.double. Its not needed for tinygo but it increases the compatibility with oficial golang toolchain.

I will separate it in another pull request.

Reply:
Thanks! Makes sense.

You can run `go test ./cgo` to test the changes locally. It may also be a good idea to add a test to testdata/cgo, to check that `C.float` etc works.